tasks.py

```diff
@@ -295,3 +295,38 @@ def write_file(full_path: Path, content: str):
                 full_path=docs_sub_dir.joinpath(md_filename),
                 content=template.format(name=basename, py_path=py_path),
             )
+
+
+@task(help={"filename": "The JSON file containing the OpenAPI schema to validate"})
+def run_swagger_validator(_, fname):
```

Comment:
Suggestion: Since the command will already start with "invoke", it sort of feels like double-trouble to say "invoke run X" and not just "invoke X". But up to you :)
```suggestion
def swagger_validator(_, fname):
```